src/components/RadioButtonGroup/RadioButtonGroup.tsx

```diff
@@ -40,34 +47,61 @@ export interface RadioButtonGroupProps extends HTMLProps<HTMLDivElement> {
    * The ref to the HTML Dom element
    */
   ref?: Ref<HTMLDivElement>;
+  /**
+   * Warning or error message, displayed in a tooltip.
```

Comment:
I know this is probably copied from another input component, but it's no longer accurate 🙈
```suggestion
   * Warning or error message, displayed below the radio buttons.
```